src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -384,6 +384,8 @@ if(BUILD_SHARED_LIBS)
     set_target_properties(${TARGET_NAME}_obj PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
 endif()
 
+ov_add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
```

Comment:
you can add a flag to `ov_add_plugin`
https://github.com/openvinotoolkit/openvino/blob/ea72f3047cb2fca24a89bed7c6069d4abb960a99/cmake/developer_package/plugins/plugins.cmake#L36

Reply:
Done